<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LIN-5CG134B5MD" failures="1" tests="1" name="Default test" time="18.678" errors="0" timestamp="2024-04-18T15:29:50 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="16.42">
    <failure type="java.lang.AssertionError" message="expected [https://magicbricks.com/vastu-consultant/sample-report] but found [https://www.magicbricks.com/propertyservices/vastu-services]">
      <![CDATA[java.lang.AssertionError: expected [https://magicbricks.com/vastu-consultant/sample-report] but found [https://www.magicbricks.com/propertyservices/vastu-services]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at MbPages.HLPageFactory.defectMsgAssertion(HLPageFactory.java:317)
at MbStepDefinition.DefectStep_Definition.report_will_be_open_on_another_page(DefectStep_Definition.java:52)
at âœ½.Report will be open on another page(file:///C:/Users/RSHARM31/.eclipse/Homeloans/src/test/resource/MbFeatures/Defect.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
